Lesson1/src/lesson1.java

```diff
@@ -1,6 +1,5 @@
-public class lesson1 {
+public class Lesson1 {
```

Comment:
please delete this file and write code in file Lesson1.java according to java naming conventions(https://www.javatpoint.com/java-naming-conventions) you need to name classes starting from capital letter for every new word in class, like CustomService (and file name should match class name, like CustomService.java)